tests/Doctrine/Tests/DBAL/DriverManagerTest.php

```diff
@@ -217,7 +240,7 @@ public function databaseUrls()
                 false,
             ),
             'URL without scheme but default PDO driver' => array(
-                array('url' => '//foo:bar@localhost/baz', 'pdo' => $pdoMock),
+                array('url' => '//foo:bar@localhost/baz', 'pdo' => true),
```

Comment:
This is a test change that I don't understand.

Reply:
@Ocramius look at the changes in `testDatabaseUrl()`. Instead of creating mock, the data provider passes a flag to the test method. It will either create the mock if PDO is loaded or will skip the data set.

Reply:
Got it 👍